### PR TITLE
Remove three unused functions

### PR DESCRIPTION
**Pull request by nsajko**

### CODE REVIEW

File: libr/core/cmd.c [hunk 1]
```diff
@@ -1322,27 +1322,6 @@ static int cmd_pipein(void *user, const char *input) {
 	return 0;
 }
 
-static void task_test(RCore *core, int usecs) {
```
Comment (ret2libc) [inline libr/core/cmd.c]: since you are removing this, please also remove the `#if 0` code in `core/cmd.c` where task_test is "used".

**Comment by radare** [comment]
@thestr4ng3r can you check this test function? Do you want it to have it in the unit tests?

**Comment by radare** [comment]
About the disasm ones... i guess its fine, but will be good to git blame to see at which point we stopped using them and if thats not introducing any regression maybe @kazarmy can answer that

**Comment by kazarmy** [comment]
sorry no time to look at this at the moment ... i prefer `#if 0` instead of removing the functions outright

**Comment by radare** [comment]
Can you update the pr according to these comments? Thanks!

**Comment by nsajko** [comment]
> Can you update the pr according to these comments? Thanks!

I will wait until @kazarmy or someone else has time to decide if the disasm.c functions are needed or should be removed.

**Comment by radare** [comment]
Just if0. He will be offline a bunch of days. We have more if0 code to review and decide probably in another pr

**Comment by nsajko** [comment]
> Just if0.

Done.

**Pull request merged**